regtests/bin/matrix.base

```diff
@@ -2003,6 +2004,16 @@
    echo "$rtst -s ST4 -w work_UPD6_U_cap -i input_UPD6_U_cap $ww3 ww3_ta1" >> matrix.body
   fi
 
+ #Test of atmosphere, ocean, and ice coupling 
+ if [ "$coup" = 'y' ]
+ then
+   echo ' ' >> matrix.body
+   echo "$rtst -s OASACM -w work_OASACM -C OASIS -f -p $mpi -n $np -o netcdf $ww3 ww3_tp2.14" >> matrix.body
+   echo "$rtst -s OASACM2 -w work_OASACM2 -C OASIS -f -p $mpi -n $np -o netcdf $ww3 ww3_tp2.14" >> matrix.body
+   echo "$rtst -s OASOCM -w work_OASOCM -C OASIS -f -p $mpi -n $np -o netcdf $ww3 ww3_tp2.14" >> matrix.body
+   echo "$rtst -s OASICM -w work_OASICM -C OASIS -f -p $mpi -n $np -o netcdf $ww3 ww3_tp2.14" >> matrix.body
+ fi
+ 
 # 365_day and 360_day calendars, no switch sharing here
 
   if [ "$calendar" = 'y' ] && [ "$shrd" = 'y' ]
```

Comment:
This does not need the "$shrd"=y as there is not an alternative dist option.

Reply:
I have modified the new regtests to specify that they are for OASIS coupling, and they will only run when dist='y'. All the dependencies are internal, i.e., all the libraries needed are already contained and compiled in the regtests without further external libraries. For that reason, I think it is a good idea to leave them in NCEP.

For the moment I leave out the modifications for calendar, as they belong to another ticket, and because imposing shrd='y' is only redundant. I can easily change it if you insist on modifying this here, though.